nss-day-cohort-47/reactive-nutshell-the-fantastic-404s
# Lm creating dashboard

Description
Adding homepage for the dashboard

Type of change
[x] New feature (non-breaking change which adds functionality)

Testing Instructions
Check code and run natively, the home tab should generate the task highlight

Checklist:
[x ] My code follows the style guidelines of this project
[x ] I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x ] I have added test instructions that prove my fix is effective or that my feature works
